### PR TITLE
Admin App: Switch to MUI

### PR DESCRIPTION
**Pull request by amiraliemami**
Reviewer: @lickem22 
Estimate: 30mins

---

## Ticket

Fixes: [422](https://idinsight.atlassian.net/browse/AAQ-422)

### Goal

- Merge the MUI refactor feature into main.

## How has this been tested?

- Follow [local dev setup](https://idinsight.github.io/aaq-core/develop/setup/) from docs and test the front-end
- Follow the [local docker-compose deployment](https://idinsight.github.io/aaq-core/deployment/quick-setup/) docs and test the front-end

## To-do before merge

- [x] Merge #136 into `feature-mui-admin`
- [x] rebase with `main` (if there are any updates).

## To do

- Add post to Updates section announcing the switch over!

## Checklist

Fill with `x` for completed. Delete any lines that are not relevant

- [x] My code follows the style guidelines of this project
- [x] I have reviewed my own code to ensure good quality
- [x] I have tested the functionality of my code to ensure it works as intended
- [x] I have resolved merge conflicts
- [x] I have updated the requirements (if applicable)

### CODE REVIEW

File: admin_app/src/app/content/edit/page.tsx [hunk 1]
```diff
@@ -0,0 +1,263 @@
...
+  );
+};
+
+const ContentBox = ({
+  content,
+  setContent,
+}: {
+  content: Content | null;
+  setContent: React.Dispatch<React.SetStateAction<Content | null>>;
+}) => {
```
Comment (lickem22) [inline admin_app/src/app/content/edit/page.tsx:86]: Nice code!

File: admin_app/package.json [hunk 2]
```diff
@@ -9,24 +9,23 @@
...
   "devDependencies": {
-    "@types/node": "^20.8.10",
+    "@types/node": "^20",
     "@types/react": "^18",
     "@types/react-dom": "^18",
-    "autoprefixer": "^10",
     "eslint": "^8",
-    "eslint-config-next": "13.5.4",
-    "postcss": "^8",
-    "prettier": "^3.0.3",
```
Comment (sidravi1) [inline admin_app/package.json:29]: Where did prettier go?
We probably need it right?

File: deployment/docker-compose/docker-compose.yml [hunk 3]
```diff
@@ -26,6 +30,10 @@ services:
     restart: always
     env_file:
       - .env
+    develop:
```
Comment (sidravi1) [inline deployment/docker-compose/docker-compose.yml:33]: Nice!

File: admin_app/tsconfig.json [hunk 4]
```diff
@@ -5,13 +5,13 @@
     "skipLibCheck": true,
     "strict": true,
     "noEmit": true,
-    "incremental": true,
     "esModuleInterop": true,
     "module": "esnext",
-    "moduleResolution": "node",
+    "moduleResolution": "bundler",
```
Comment (sidravi1) [inline admin_app/tsconfig.json:10]: I don't understand all the implications of this. Do you guys?

**Review by sidravi1 — COMMENTED** [review]
Exciting to see this going in! 
Nice work @amiraliemami - that was some serious hustling.

File: admin_app/next.config.mjs [hunk 5]
```diff
@@ -0,0 +1,8 @@
+/* @type {import('next').NextConfig} */
+const nextConfig = {
+  experimental: {
+    missingSuspenseWithCSRBailout: false,
```
Comment (sidravi1) [inline admin_app/next.config.mjs:4]: I somehow missed this in previous PRs. Why are we doing this? And is it possible to fix the root issue?

https://nextjs.org/docs/messages/missing-suspense-with-csr-bailout

File: admin_app/package.json (hunk 2 above)
Comment (amiraliemami) [reply admin_app/package.json:29]: Not sure, maybe something went wrong with my rebase? Please add it back if you can

File: admin_app/tsconfig.json (hunk 4 above)
Comment (amiraliemami) [reply admin_app/tsconfig.json:10]: I don't either

**Pull request merged**